### PR TITLE
Clean up `.Rbuildignore`

### LINKED ISSUE #886: Root directory must be clean of non-standard files and directories (use `.Rbuildignore` to control this)

### PR DESCRIPTION
**Pull request by edelarua**
Closes #886

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -4,16 +4,11 @@
...
 
-# design
-design/*
-
 # doc
-CODE_OF_CONDUCT.md
+^CODE_OF_CONDUCT\.md$
 ^docs$
 ^LICENSE\.md$
-README.*
```
Comment (shajoezhu) [inline .Rbuildignore:14]: readme.md as well can be ignored?

**Review by Melkiades — APPROVED** [review]
Lgtm! Thanks Emily :)

File: .Rbuildignore (hunk 1 above)
Comment (Melkiades) [reply .Rbuildignore:14]: I do not think so. With this it should be tracked again correctly

Comment (edelarua) [reply .Rbuildignore:14]: That's correct - `README.md` is a recognized file and does not need to be ignored.

**Pull request merged**